service/src/test/java/bio/terra/cbas/controllers/TestRunsApiController.java

```diff
@@ -76,6 +77,8 @@ class TestRunsApiController {
   @Autowired private ObjectMapper objectMapper;
   @MockBean private SamService samService;
 
+  @MockBean private MeterRegistry meterRegistry;
```

Comment:
My hunch is that if you use `@Autowired` here instead, the framework may provide an instance of `SimpleMeterRegistry` which shouldn't return `null` from the `Counter.builder` chain.

If this doesn't work out of the box, then I think it still _should_ be possible to coerce the framework to inject a `SimpleMeterRegistry` here, and this would let you sidestep the null check on counter.